server/app.py

```diff
@@ -44,15 +44,19 @@ def create_app():
     db.init_app(app)
     api = Api(app)
 
-    api.add_resource(UserResource, '/users')
-    api.add_resource(ChefResource, '/chefs', '/chefs/<id>')
-    api.add_resource(LoginResource, '/users/login')
-    api.add_resource(MealItemResource, '/meal_items', '/meal_items/<id>')
-    api.add_resource(LogoutResource, '/users/logout')
-    api.add_resource(CuisineResource, '/cuisines')
-    api.add_resource(NotificationResource, '/notifications')
-    api.add_resource(StripeResource, '/create-payment-intent')
-    api.add_resource(OrderResource, '/orders', '/orders/<id>')
+    @app.route('/')
+    def index():
```

Comment:
@app.route('/')
    @app.route('/<path1>')
    @app.route('/<path1>/<path2>')
    @app.route('/<path1>/<path2>/<path3>')
    @app.route('/<path1>/<path2>/<path3>/<path4>')
    def index(**kwargs):
        return app.send_static_file('index.html')
replace with this and you'll be able to reload on any front end route.

Reply:
What this is doing is saying no matter what variables come after the `/` still send index.html
For example, `@app.route('/<path1>/<path2>')` means even ifs `'/anystring/anystring2/'` still serve index.html

Reply:
Thanks Moshe! This was what I was looking for!